classes/class-suggested-tasks.php

```diff
@@ -633,6 +633,9 @@ public function suggested_task_action() {
 				// Insert an activity.
 				$this->insert_activity( $task_id );
 				$updated = true;
+
+				// Allow other classes to react to the completion of a suggested task.
+				\do_action( 'progress_planner_task_dismissed', $task_id );
```

Comment:
Instead of doing this, it would probably be better if we do it _after_ the `switch`, and use an action-name like `"progress_planner_task_action_$action"`.
This way it will be possible to allow hooking in more action if and when it's necessary.